presto-tpcds/src/main/resources/META-INF.services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+com.facebook.presto.tpcds.TpcdsPlugin
```

Comment:
Since you have `<packaging>presto-plugin</packaging>` in the pom.xml file, you don't need this file